src/search/heuristics/hm_heuristic.cc

```diff
@@ -269,7 +275,7 @@ class HMHeuristicFeature : public plugins::TypedFeature<Evaluator, HMHeuristic>
         document_title("h^m heuristic");
 
         add_option<int>("m", "subset size", "2", plugins::Bounds("1", "infinity"));
-        Heuristic::add_options_to_feature(*this);
+        Heuristic::add_options_to_feature(*this, "hm");
```

Comment:
We had some discussions (I don't remember where and who participated) that this injection of default values into the base class will eventually become problematic when we switch to loggers. At this point, I just want to be sure that we're all aware of this and that there won't be surprises if we end up essentially undoing this change at a later point.